<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client</class>
 <widget class="QDialog" name="client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户端</string>
  </property>
  <property name="toolTip">
   <string>客户端</string>
  </property>
  <property name="whatsThis">
   <string>客户端</string>
  </property>
  <widget class="QTableWidget" name="atableWidget">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>300</y>
     <width>400</width>
     <height>250</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>250</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>250</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="messagetextBrowser">
   <property name="geometry">
    <rect>
     <x>803</x>
     <y>80</y>
     <width>331</width>
     <height>91</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>91</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>80</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数据显示区：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>50</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>数据发送区：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientMessagelineEdit">
   <property name="geometry">
    <rect>
     <x>768</x>
     <y>50</y>
     <width>108</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="clientSendpushButton">
   <property name="geometry">
    <rect>
     <x>882</x>
     <y>51</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cCleanpushButton">
   <property name="geometry">
    <rect>
     <x>968</x>
     <y>51</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectpushButton">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>20</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnectpushButton">
   <property name="geometry">
    <rect>
     <x>1050</x>
     <y>50</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>断开</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientPortlineEdit">
   <property name="geometry">
    <rect>
     <x>924</x>
     <y>20</y>
     <width>108</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>8085</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="clientIPlineEdit">
   <property name="geometry">
    <rect>
     <x>768</x>
     <y>20</y>
     <width>108</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.1.127</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>882</x>
     <y>20</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QLabel" name="cStatuslabel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>120</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>180</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>基站数目：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="basecomboBox">
   <property name="geometry">
    <rect>
     <x>756</x>
     <y>180</y>
     <width>36</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="recordpushButton">
   <property name="geometry">
    <rect>
     <x>798</x>
     <y>181</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>开始记录数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stoprecordpushButton">
   <property name="geometry">
    <rect>
     <x>884</x>
     <y>181</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>停止记录数据</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>180</y>
     <width>161</width>
     <height>101</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;请输入基站【1】坐标&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;An【1】（450，685，0）&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>270</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确认输入当前坐标值</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recordpushButton_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>开始进行位置估计</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clientloadpicture">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>载入图片并进行障碍物建模</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>220</y>
     <width>251</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>基站序号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="basecomboBox_2">
        <property name="currentText">
         <string>1</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>X坐标</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="text">
         <string>450</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Y坐标</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="text">
         <string>685</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Z坐标</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>label_2</zorder>
  <zorder>clientMessagelineEdit</zorder>
  <zorder>clientSendpushButton</zorder>
  <zorder>cCleanpushButton</zorder>
  <zorder>connectpushButton</zorder>
  <zorder>disconnectpushButton</zorder>
  <zorder>clientPortlineEdit</zorder>
  <zorder>label</zorder>
  <zorder>clientIPlineEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>cStatuslabel</zorder>
  <zorder>label_5</zorder>
  <zorder>basecomboBox</zorder>
  <zorder>recordpushButton</zorder>
  <zorder>stoprecordpushButton</zorder>
  <zorder>atableWidget</zorder>
  <zorder>messagetextBrowser</zorder>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
  <zorder>textEdit</zorder>
  <zorder>basecomboBox_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>pushButton</zorder>
  <zorder>textEdit_2</zorder>
  <zorder>recordpushButton_2</zorder>
  <zorder>clientloadpicture</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
